tests/integration/adapter-identifier-tests.js

```diff
@@ -0,0 +1,308 @@
+import set from 'lodash-es/set';
+
+function adapterIdentifierTests() {
```

Comment:
The general id here is if we change any of the identifiers, we either introduced a breaking change or a bug.  In writing these tests, I did spot existing bug - there could be more.